### PR TITLE
Add chapter about our interview process.

### PR DESCRIPTION
**Pull request by aquamatthias**

### CODE REVIEW

File: hiring.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- **Technical Interview**
+
+  In this 1-hour video call, we want to learn your technical expertise.
+  Ideally, this is a conversation about technologies, architecture, and good practice.
+
+- **Team Interview**
+
+  We collaborate in teams and think it is essential to feel engaged with the people you work with.
+  In this interview, you have the chance to get to know the team and vice versa.
```
Comment (TheCatLady) [inline hiring.md:27]:
```suggestion
### Initial Contact

This interview is a 30-minute video call for us to get to know each other.

### Coding Challenge

The coding challenge is "homework" that should not take longer than 4 hours.

You will be asked to submit a pull request on GitHub with your changes, so we can discuss them during a 20-minute follow-up call.

We want to see how you think and approach challenges.

### Technical Interview

The goal of this 1-hour video call is to test your technical knowledge and expertise.

Please be prepared to discuss technologies, architecture, and best practices.

### Team Interview

We collaborate in teams and strongly believe that it is essential to feel connected and engaged with your teammates.

This interview is a chance for you to get to know the team, and vice versa.
```
Using sub-headings is best, as it allows for you to specific link to parts of the document/page.

File: hiring.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Hiring
+
+If you are interested in a job at Some Engineering Inc., please send your application to [hr@some.engineering](mailto:hr@some.engineering).
```
Comment (TheCatLady) [inline hiring.md:3]:
```suggestion
If you are interested in a job at Some Engineering Inc., please send your application to [hr@some.engineering](mailto:hr@some.engineering).
```
Do we have an application, or are candidates simply submitting a resume?

File: hiring.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Hiring
```
Comment (TheCatLady) [inline hiring.md:1]:
```suggestion
# Interview Process
```
This file would also ideally be renamed.

File: hiring.md (hunk 2 above)
Comment (aquamatthias) [reply hiring.md]: Simply submitting a resume and the open position is probably sufficient.

File: interview-process.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Interview Process
+
+If you are interested in a job at Some Engineering Inc., please send your application to [hr@some.engineering](mailto:hr@some.engineering).
+
+## Interview process for engineers
+
+The interview process for engineers usually has those steps:
+
+### Initial Contact
```
Comment (lloesche) [inline interview-process.md:9]: Intro Call

File: interview-process.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# Interview Process
+
+If you are interested in a job at Some Engineering Inc., please send your application to [hr@some.engineering](mailto:hr@some.engineering).
+
+## Interview process for engineers
+
+The interview process for engineers usually has those steps:
```
Comment (TheCatLady) [inline interview-process.md:7]:
```suggestion
## Engineering Interview Process

For engineering candidates, there are four stages to the interview process.
```

File: interview-process.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+# Interview Process
+
+If you are interested in a job at Some Engineering Inc., please send your application to [hr@some.engineering](mailto:hr@some.engineering).
```
Comment (TheCatLady) [inline interview-process.md:3]:
```suggestion
If you are interested in a job at Some Engineering Inc., please email your resume/CV to [hr@some.engineering](mailto:hr@some.engineering).
```

**Pull request merged**